WalletWasabi.Gui/Tabs/WalletManager/LoadWallets/LoadWalletViewModel.cs

```diff
@@ -40,17 +44,33 @@ public LoadWalletViewModel(WalletManagerViewModel owner, LoadWalletType loadWall
 			Owner = owner;
 			Password = "";
 			LoadWalletType = loadWalletType;
-			Wallets = new ObservableCollection<LoadWalletEntry>();
 
-			this.WhenAnyValue(x => x.SelectedWallet)
-				.Subscribe(_ => TrySetWalletStates());
+			RootList = new SourceList<WalletViewModelBase>();
+			RootList
+				.Connect()
```

Comment:
Maybe the first one should be in the same line?
```suggestion
			RootList.Connect()
```